File: Solutions/Stareat0707/Game Shop/Game Shop, Part 2/Weapon.cpp
#include "Weapon.h"

Weapon::Weapon(std::string name, std::string description, int weight, int value, int damage) : Item(name, description, weight, value), damage(damage)
{
}

void Weapon::Describe()
{
	Item::Describe();
	std::cout << "Damage        = " << damage << std::endl;
}

File: Solutions/whiteWeed/Game Shop/4/Weapon.java
/**
 * 
 */
package whiteWeed.gameShop;

/**
 * <pre>
 * whiteWeed.gameShop
 *   |_Weapon
 * 
 * 1. 개요 : 
 * 2. 작성일 : 2017. 5. 29.
 * </pre>
 * 
 * @author				:  Weed
 * @version				:  1.0
 */
public class Weapon extends Item {
	private int damage; //ㅁㄴㅇㄹ
	
	public Weapon(String name, String description, int weight, int value, int damage){
		super(name, description, weight, value);
		this.damage = damage;
	}
	
	public void describe(){
		super.describe();
		System.out.println("Damage        = " + this.damage);
	}
}
